packages/react-app-revamp/config/wagmi/custom-chains/evmosTestnet.ts

```diff
@@ -1,9 +1,10 @@
-import { Chain } from "wagmi";
+import { Chain } from "@rainbow-me/rainbowkit";
 
 export const evmosTestnet: Chain = {
   id: 9000,
   name: "EvmosTestnet",
```

Comment:
should we update this one to the same capitalization rules as evmosMainnet since we do that in that file in this PR as well?